### ISSUE TITLE
NTR: 'leukocyte adhesion to vascular endothelial cell’

### ISSUE THREAD

**Issue by BarbaraCzub** [geneontology/go-ontology#12276]
Dear Editors,
I wish to request a new GO term for annotating paper PMID:23897866.
 
It is demonstrated in Figure 6B and C in this paper that leukocytes adhere to vascular endothelial cells, following regulation of this process by hsa-miR-92a-3p, and stimulation with TNFalpha.

In wish to annotate this as: 

Object:			        RNAC: URS00003768C5_9606 (hsa-miR-92a-1-3p)   
GO term:	                        Negative regulation of leukocyte adhesion to vascular endothelial cell
Evidence:		        IMP   
Annotation Extension:	part_of GO:0042044 (cellular response to tumour necrosis factor)

Therefore, I will also be requesting regulation terms for ‘leukocyte adhesion to vascular endothelial cell’ via TermGenie.

I will look forward to hearing from you with regard to this NTR.

Thank you,
Barbara

#12249 
https://github.com/obophenotype/cell-ontology/issues/420 

@rachhuntley 
@RLovering

**Comment by BarbaraCzub** [comment]
Forgot to add:

DbxREFs: GOC:BHF, GOC:BHF_miRNA, GOC:bc

**Comment by BarbaraCzub** [comment]
To further support the NTR, I wish to add that the requested term would also be used to annotate findings presented in Figure 5 of PMID:22267480.

DbxREFs: GOC:BHF, GOC:BHF_miRNA, GOC:bc

@rachhuntley 
@RLovering

**Comment by BarbaraCzub** [comment]
Following a team discussion re: findings presented in Figure 5 of PMID:22267480, I wish to request another new term, specifically: 
'leukocyte adhesion to arterial endothelial cell'. 
Presumably this would become a child of the 'leukocyte adhesion to vascular endothelial cell', which I requested initially.

I will look forward to hearing from you with regard to both NTRs.

Thank you,
Barbara

DbxREFs: GOC:BHF, GOC:BHF_miRNA, GOC:bc

@rachhuntley 
@RLovering

**Comment by addiehl** [comment]
Hi Barbara,

There are existing terms in the GO that describe these processes:

'leukocyte tethering or rolling' (GO:0050901) defined as "Transient adhesive interactions between leukocytes and endothelial cells lining blood vessels. Carbohydrates on circulating leukocytes bind selectins on the vessel wall causing the leukocytes to slow down and roll along the inner surface of the vessel wall. During this rolling motion, transitory bonds are formed and broken between selectins and their ligands. Typically the first step in cellular extravasation (the movement of leukocytes out of the circulatory system, towards the site of tissue damage or infection"

'leukocyte adhesive activation' (GO:0050902) defined as "The activation of loosely bound or rolling leukocytes by signals displayed on blood vessel endothelial cells, which is typically the second step in cellular extravasation."

'leukocyte activation-dependent arrest' (GO:0050903) defined as "The formation of an integrin-dependent strong adhesive bond between leukocytes and blood vessel endothelial cells which is dependent on prior activation of the leukocyte and leads to the firm attachment of the leukocyte to the endothelial surface, typically the third step in cellular extravasation."

'diapedesis' (GO:0050904) defined as "The passage of a leukocyte between the tight junctions of endothelial cells lining blood vessels, typically the fourth and final step of cellular extravasation."

These terms are all part_of to 'cellular extravasation' (GO:0045123) defined as "The migration of a leukocyte from the blood vessels into the surrounding tissue."  All these terms specify vascular endothelial cells (AKA  "endothelial cells lining blood vessels" and "blood vessel endothelial cells") as part of their definitions.

Based on the data in figure 3, and the fact that the authors specifically mention that miR-92a targets integrin alpha-5 in endothelial cells, I would argue that 'leukocyte activation-dependent arrest' is the appropriate term to annotate the data, per its definition.

However, since it's always great to add new terms to the GO and increase the complexity of the polyhierarchy and the availability of partially redundant terms, I would suggest we add 'leukocyte adhesion to vascular endothelial cell' as a subtype of 'leukocyte cell-cell adhesion', and make both 'leukocyte tethering or rolling' and 'leukocyte activation-dependent arrest' children of this term, as opposed to being direct children of 'leukocyte activation-dependent arrest'.  Then at least the annotations to these subtypes of ''leukocyte adhesion to vascular endothelial cell' will propagate to this new term, immediately increasing its utility in term enrichment analysis.

The 'leukocyte adhesion to arterial endothelial cell' term can go in as you suggested.

Best regards,
Alex

**Comment by BarbaraCzub** [comment]
Hi Alex,

Thank you for clarifying this GO branch for me!

I see your point about the 'leukocyte activation-dependent arrest', however, based on the data from paper PMID:23897866 and the TAS they make about integrin being a target of miR-92a, I would probably only be able to make such an annotation using the IC evidence code... (And I would also need to request regulation terms first, if I wanted to annotate Figure 6). 

My annotation(s) would be more accurate, if I could say 'leukocyte adhesion to vascular/arterial endothelial cell' as this would unquestionably reflect the experimental evidence provided within the paper(s) being annotated.

And, I agree that it would be a good idea to add this term in as a more specific parent of 'leukocyte tethering or rolling' and 'leukocyte activation-dependent arrest' (atm they are children of the general 'leukocyte cell-cell adhesion' term).

Still, in view of what you suggested, I'll need to discuss this issue with Rachael before creating the intended annotations.

Thank you,
Barbara

@rachhuntley 
@RLovering

**Comment by ukemi** [comment]
Here is what I added and I moved the terms as Alex suggested. Alex, I added your dbxref as well. Thanks for the advice. Let me know if things look incorrect.
-D
 [Term]
+id: GO:0061756
+name: leukocyte adhesion to vascular endothelial cell
+namespace: biological_process
+def: "The attachment of a leukocyte to vascular endothelial cell via adhesion molecules." [GOC:add, GOC:bc, GOC:BHF, GOC:BHF_miRNA, PMID:23897866]
+is_a: GO:0007159 ! leukocyte cell-cell adhesion
+created_by: dph
+creation_date: 2016-02-29T16:42:11Z
+
+[Term]
+id: GO:0061757
+name: leukocyte adhesion to arterial endothelial cell
+namespace: biological_process
+def: "The attachment of a leukocyte to an arterial endothelial cell via adhesion molecules." [GOC:add, GOC:bc, GOC:BHF, GOC:BHF_miRNA, PMID:22267480]
+is_a: GO:0061756 ! leukocyte adhesion to vascular endothelial cell
+created_by: dph
+creation_date: 2016-02-29T16:49:38Z
